documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -59,7 +59,7 @@ NOTE: The value of `log.message.format.version` and `inter.broker.protocol.versi
 ====
 Changing the `kafka.version` ensures that all brokers in the cluster will be upgraded to start using the new broker binaries.
 During this process, some brokers are using the old binaries while others have already upgraded to the new ones.
-Leaving the `inter.broker.protocol.version` unchanged ensures that the brokers can continue to communicate with each other throughout the upgrade.
+Leaving the `inter.broker.protocol.version` set to the current version ensures that the brokers can continue to communicate with each other throughout the upgrade.
```

Comment:
The current here should mean "to keep the current setting" and not "set it according to current Kafka version". I think this can be a bit confusing. Maybe we can make it more clear?